recipe.yml

```diff
@@ -1,11 +1,11 @@
-# Warning: only edit this file inside the images git repository.
-# If you are looking at this file in your root directory on your computer, editing it does nothing.
+# Warning: Only edit this file inside the uBlue image's git repository.
```

Comment:
This warning is useless, as now the recipe resides in an immutable directory.

Reply:
@EinoHR Very good point, removing the recipe.yml edit warning since it's no longer in a mutable directory. :)